src/LibraryViewExtensionWebView2/ViewExtension.cs

```diff
@@ -37,7 +38,17 @@ public void Loaded(ViewLoadedParams viewLoadedParams)
                 controller = new LibraryViewController(viewLoadedParams.DynamoWindow, viewLoadedParams.CommandExecutive, customization);
                 (viewLoadedParams.DynamoWindow.DataContext as DynamoViewModel).PropertyChanged += handleDynamoViewPropertyChanges;
             }
-            
+            viewParams.CurrentWorkspaceChanged += ViewParams_CurrentWorkspaceChanged;
+        }
+
+        private void ViewParams_CurrentWorkspaceChanged(IWorkspaceModel workspace)
+        {
+            var type = workspace switch
+            {
+                HomeWorkspaceModel _ => "home",
```

Comment:
I see you defined the next line in librarie.js
`export type HostingContextType = "home"|"custom"|"none"`
What happens if is set to "none"?

Reply:
custom and none do nothing special at the moment - library is rendered as before my prs.